### ISSUE TITLE
Font style popover in Epiphany missing padding

### ISSUE THREAD

**Issue by kdwk** [elementary/stylesheet#1087]
## What Happened

![Screenshot from 2021-09-05 21-04-29](https://user-images.githubusercontent.com/39268094/132127987-d1b8c232-03ac-4d66-b6c2-974906b5d619.png)

## Expected Behavior

Should be consistent with system style

## Steps to Reproduce

1. Open Preferences-> Appearance-> Reader Mode
2.
3.

## Platform Information

Flatpak org.gnome.Epiphany 3.38.6
elementary OS 6 Odin

**Comment by danrabbit** [comment]
Yeah, for whoever gets to this first this is likely from a LibHandy widget so it would be easiest to test against Hdy.Demo

**Comment by Chris-Mingay** [comment]
Straight up I am a complete beginner with gtk but I am interested in helping with some of the more cosmetic fixes in elementary os.

From your previous comment @danrabbit I take it that instead of having to test this against Epiphany I can get away with testing it against "Hdy.Demo" of LibHandy?

I've built the example keypad from the LibHandy repo and I see reference to "hdy-demo" in the examples folder however I'm not so sure how to actually "run" Hdy.Demo, can you point me in the right direction?

Thanks

**Comment by cassidyjames** [comment]
@Chris-Mingay on elementary OS, you can just install it directly from Terminal with:

```sh
sudo apt install handy-1-examples
```

Then run it from Terminal or the Applications Menu by typing:

```sh
handy-1-demo
```

**Comment by Chris-Mingay** [comment]
Thanks @cassidyjames don't want to take up too much of your time so this issue might be better suited to someone else now but when trying to run `handy-1-demo` I get

```
handy-1-demo: symbol lookup error: handy-1-demo: undefined symbol: hdy_style_manager_get_dark, version LIBHANDY_1_0
```

Again, still happy to be the one to look in to this when/if I get set up!

**Comment by danirabbit** [comment]
This has been fixed in #1245
